compile.c

```diff
@@ -4130,12 +4130,11 @@ compile_keyword_arg(rb_iseq_t *iseq, LINK_ANCHOR *const ret,
             struct rb_callinfo_kwarg *kw_arg =
                 rb_xmalloc_mul_add(len, sizeof(VALUE), sizeof(struct rb_callinfo_kwarg));
 	    VALUE *keywords = kw_arg->keywords;
-	    int i;
 	    kw_arg->keyword_len = len;
 
 	    *kw_arg_ptr = kw_arg;
 
-	    for (i=0; node != NULL; i++, node = node->nd_next->nd_next) {
+	    for (int i=0; node != NULL; i++, node = node->nd_next->nd_next) {
```

Comment:
I don't know what is consistent but I prefer `int i = 0` with spaces.

Reply:
@ioquatix 
Sorry, added space(095cb225bc0d6387decd7d0044acdf860e42a3d2)